src/main/webapp/components/directives/cache/remotestore/remotestore.html

```diff
@@ -0,0 +1,136 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="form-horizontal">
+      <div class="form-group">
```

Comment:
Here I would certainly use ng-repeat for all this form groups! It will reduce repetition a lot.